docs/auth.md

```diff
@@ -1 +1 @@
-**We've moved our documentation!** Check out the latest [authentication content](https://docs.influxdata.com/chronograf/latest/administration/security-best-practices/#chronograf-with-oauth-2-0-authentication) on InfluxData's [main docs site](https://docs.influxdata.com/chronograf/latest/).
+**We've moved our documentation!** Check out the latest [authentication content](https://docs.influxdata.com/chronograf/v1.4/administration/managing-security/#oauth-2-0-providers-with-jwt-tokens) on InfluxData's [main docs site](https://docs.influxdata.com/chronograf/latest/).
```

Comment:
let's use `/latest` instead of `/v1.4` and otherwise good to go -- thanks!

Reply:
👍